docs/howto/build-guides/platforms.rst

```diff
@@ -0,0 +1,169 @@
+Build platform-specific charms
+==============================
+
+A ``charmcraft.yaml`` defines where charms are built and where charms are run
+using the ``platforms``, ``base``, and ``build-base`` keywords.
```

Comment:
Again being pedantic, but we don't show usage of `build-base` in this doc.

Reply:
Removed for now, and added an explicit documentation notice to https://github.com/canonical/charmcraft/pull/2080/files#r1919338151